Simplenote/SPNoteEditorViewController+Extensions.swift

```diff
@@ -8,50 +8,58 @@ extension SPNoteEditorViewController {
     ///
     @objc
     var isShowingHistory: Bool {
-        return historyCardViewController != nil
+        return historyViewController != nil
     }
 
     /// Shows note history
     ///
     @objc
     func showHistory() {
         let loader = SPHistoryLoader(note: currentNote)
-        let cardViewController = newHistoryViewController(with: loader, delegate: self)
+        let viewController = newHistoryViewController(with: loader, delegate: self)
 
-        historyCardViewController = cardViewController
+        historyViewController = viewController
         historyLoader = loader
 
-        cardViewController.present(from: self)
+        let transitioningManager = SPCardTransitioningManager()
+        transitioningManager.observer = self
+        historyTransitioningManager = transitioningManager
+
+        present(viewController, with: transitioningManager)
     }
 
-    private func newHistoryViewController(with loader: SPHistoryLoader, delegate: SPNoteHistoryControllerDelegate) -> SPCardViewController {
+    private func newHistoryViewController(with loader: SPHistoryLoader,
+                                          delegate: SPNoteHistoryControllerDelegate) -> UIViewController {
```

Comment:
Q: Do you think it'd be cleaner to return a `SPNoteHistoryViewController`, and allow the caller to set the delegate?

Why: One less parameter, you get to simply `return SPHistoryViewController(...`, and it's cleaner!

Reply:
Delegate is actually for the controller (not view controller) and is created inside this method. 
But isn't it nice to return a plain UIViewController and hide the actual creation process as the caller doesn't really care what class it is as long as it's UIViewController and can be presented :-)